velox/connectors/hive/HiveConfig.h

```diff
@@ -206,6 +206,12 @@ class HiveConfig {
   static constexpr const char* kS3UseProxyFromEnv =
       "hive.s3.use-proxy-from-env";
 
+  /// Timestamp unit for Parquet read.
+  static constexpr const char* kParquetReadTimestampUnit =
+      "hive.parquet.reader.timestamp-unit";
+  static constexpr const char* kParquetReadTimestampUnitSession =
+      "hive.parquet.reader.timestamp_unit";
```

Comment:
Is this something already existing in Presto Java?  If not, my feeling is this should be format agnostic.

Reply:
I think not. Presto handles millisecond and Spark handles microsecond, so we need this configuration.

> If not, my feeling is this should be format agnostic.

I apologize for not understanding this point. Could you clarify a bit what needs to be changed? Thanks!

Reply:
I think we can remove the `parquet` part from it (e.g. `hive.reader.timestamp-unit`) and make it applied to all file formats (seems you already doing so).

Reply:
Thanks for the pointer. Updated as suggested.